Document/0x05i-Testing-Code-Quality-and-Build-Settings.md

```diff
@@ -23,7 +23,9 @@ The v2 signature, which is supported by Android 7.0 and above, offers improved s
 
 Make sure that the release build has been signed via both the v1 and v2 schemes and that the code-signing certificate in the APK belongs to the developer.
 
-APK signatures can be verified with the `apksigner` tool.
+APK signatures can be verified with the `apksigner` tool. 
+
+For Ubuntu, you can download `apksigner` tool [HERE](https://launchpad.net/ubuntu/cosmic/amd64/apksigner/0.8-1)

```

Comment:
Hi, is it not a better idea to refer to https://developer.android.com/studio/command-line/apksigner for getting apksigner?

Reply:
I've tried 2 ways:
1. Installed Android Studio on my Ubuntu. It's a bit broken because it needs to install SDK tools as well, and it took a lot of time. It didn't give me apksigner (?!) . And one more problem is, I always have an issue with my adb module :(
2. Installed all Android-tools by using "apt install [android-tools]", and apksigner couldn't be found as well

IMO, it's better if we can just install the tool we want. For Ubuntu, I found that link which helped me to get rid of the issue. 

Cheers!

Reply:
For both issues i can help. The problem with this solution that tools will run out of sync. Have you checked https://stackoverflow.com/questions/40004884/cant-find-apksigner-executable-to-manually-sign-apk ?

Reply:
hey, sorry for the delay. I got my problem resolved. I found the `apksigner` under /Android/Sdk/build-tools/28.0.2. Could be my bad, didn't check it precisely.

Reply:
Can you update the pr accordingly or can we close it?

Reply:
We can close it :)